backend/src/main/java/de/codexbella/ContentController.java

```diff
@@ -15,11 +15,11 @@
    private final ContentService contentService;
 
    @GetMapping("/search/{searchTerm}")
-   public List<ShowSearchData> searchForShow(@RequestParam(defaultValue = "en-US") String language, @PathVariable String searchTerm) {
+   public List<ShowSearchData> searchForShows(@RequestParam(defaultValue = "en-US") String language, @PathVariable String searchTerm) {
       return contentService.searchForShows(language, searchTerm);
    }
-   @GetMapping("/getshow/{id}")
-   public Show searchForSingleShow(@RequestParam(defaultValue = "en-US") String language, @PathVariable int id) {
-      return contentService.searchForSingleShow(language, id);
+   @GetMapping("/addshow/{id}")
+   public Show addShow(@RequestParam(defaultValue = "en-US") String language, @PathVariable int id) {
```

Comment:
This method and the corresponding method in the `ContentService` are called `addShow`, but nothing is added, but a `Show` is retrieved.
If something was actually added, the mapping should be a `@PostMapping`.